Dockerfile

```diff
@@ -1,27 +1,4 @@
-FROM openjdk:8-jdk
-
-# install aws-cli
-RUN apt-get update && apt-get install -y python-is-python3 gettext-base python3-venv && \
-    wget -nv "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" && \
-    unzip awscli-bundle.zip && \
-    ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
-
-WORKDIR /usr/lib/java
-
-ENV GRAILS_VERSION 3.2.8
-
-RUN wget https://github.com/grails/grails-core/releases/download/v${GRAILS_VERSION}/grails-${GRAILS_VERSION}.zip && \
-    unzip grails-${GRAILS_VERSION}.zip && \
-    rm -rf grails-${GRAILS_VERSION}.zip && \
-    ln -s grails-${GRAILS_VERSION} grails
-
-ENV GRAILS_HOME /usr/lib/java/grails
-ENV JAVA_HOME /usr/local/openjdk-8
-ENV PATH $GRAILS_HOME/bin:$PATH
-
-RUN mkdir /app
-COPY . /app
-WORKDIR /app
-RUN grails compile
-EXPOSE 8080
-CMD grails run-app
\ No newline at end of file
+FROM eclipse-temurin:17.0.5_8-jre-focal
```

Comment:
There is a 17.0.11_9-jre-focal image available, not worth fixing for this PR but probably worth its own JIRA ticket for us to do at some point (though once we get on AWS we can move to the latest Java instead of being stuck on 17).